server/routerlicious/packages/services/src/tenant.ts

```diff
@@ -63,12 +62,18 @@ export class TenantManager implements core.ITenantManager {
             token: fromUtf8ToBase64(`${credentials.user}`),
         };
         const defaultHeaders = {
-            "Authorization": getAuthorizationTokenFromCredentials(credentials),
-            // TODO: all requests from this GitManager will have the same correlation id. Is this intended behavior?
-            "x-correlation-id": getCorrelationId() || uuid(),
+            Authorization: getAuthorizationTokenFromCredentials(credentials),
```

Comment:
Curious, any reason to remove the quotes here?

Reply:
It didn't pass eslint... showed "Properties shouldn't be quoted as all quotes are redundant", but it should be the same without quotes.

Reply:
Got it. Makes sense!